### PR TITLE
Discard image when file limit exceeds. 

### PR DESCRIPTION
**Pull request by ShivamPokhriyal**
Jira: https://dimagi-dev.atlassian.net/browse/SAAS-11633

Product Note: Show an error dialog when file of size more than 15 MB is selected by the user.

### CODE REVIEW

File: app/src/org/commcare/activities/FormEntryActivity.java [hunk 1]
```diff
@@ -400,6 +409,8 @@ private void processChooserResponse(Intent intent) {
             Toast.makeText(FormEntryActivity.this,
                     Localization.get("form.attachment.invalid"),
                     Toast.LENGTH_LONG).show();
+        } else if (FileUtil.isFileTooLargeToUpload(getContentResolver(), media)) {
```
Comment (shubham1g5) [inline app/src/org/commcare/activities/FormEntryActivity.java:412]: this seems to be check on original file again instead of the scaled image

**Review by shubham1g5 — CHANGES_REQUESTED** [review]

File: app/src/org/commcare/activities/FormEntryActivity.java (hunk 1 above)
Comment (ShivamPokhriyal) [reply app/src/org/commcare/activities/FormEntryActivity.java:412]: yeah this is for audio and video. We only compress images, so the check is on the original file.

Comment (shubham1g5) [reply app/src/org/commcare/activities/FormEntryActivity.java:412]: Makes sense

**Review by shubham1g5 — DISMISSED** [review]

**Review by shubham1g5 — APPROVED** [review]
Thanks! This looks much cleaner :)

**Comment by ShivamPokhriyal** [comment]
I'm merging this without waiting for the tests result since they're taking too long. 
I'll be running tests on different devices along with QA.

**Comment by dimagimon** [comment]
[33% (0.0%) vs master 33%](https://jenkins.dimagi.com/job/commcare-android-pr-job-builder/462255/)

**Pull request merged**